mageFile.go

```diff
@@ -0,0 +1,139 @@
+//go:build mage
+// +build mage
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/magefile/mage/mg"
+	"github.com/magefile/mage/sh"
+)
+
+func Lint() error {
+	if err := sh.RunV("golangci-lint", "run", "--timeout", "3m"); err != nil {
+		return err
+	}
+
+	if err := sh.RunV("go", "mod", "tidy"); err != nil {
+		return err
+	}
+
+	return sh.RunV("git", "diff", "--exit-code")
+}
+
+func Test() error {
+	return sh.RunV("go", "test", "./...", "-race")
+}
+
+func BuildLocal() error {
+	ldFlags := generateLDFlags()
+
+	fmt.Println(ldFlags)
+	return sh.RunV("go", "build", "-trimpath", "-ldflags", ldFlags, "-o", "apko", "./cmd/apko/main.go")
+}
+
+func KoApply() error {
+	os.Setenv("LDFLAGS", generateLDFlags())
+
+	return sh.RunV("ko", "apply", "-BRf", "config/")
```

Comment:
nit: we expand out `-B` (`--base-import-paths`) et al below.

Reply:
expanded to make it clear